install/kubernetes/helm/istio/values.yaml

```diff
@@ -405,6 +405,10 @@ global:
     # have a shared root CA for this model to work.
     enabled: false
 
+    # Should be set to the name of the cluster this installation will run in. This is required for sidecar injection
+    # to properly label proxies
+    clusterName: ""
```

Comment:
see my comment above - this must match the name used in the 'registry' config, I don't think it can be a generated UUID ( without making it hard to configure the registries). And there are plans to use the cluster name in telemetry, policy,etc.